src/stories/components/ChecDropdown.stories.mdx

```diff
@@ -44,10 +44,13 @@ Todo list
           default: text('Placeholder', ''),
         },
         errored: {
-          default: boolean('Errored', false)
+          default: boolean('Errored', false),
         },
         disabled: {
-          default: boolean('Disabled', false)
+          default: boolean('Disabled', false),
+        },
+        loading: {
+          default: boolean('Loading', false),
```

Comment:
What do you think of having the loading option first on the list and initially loading by default?

Reply:
The loading indicator appears on the bottom of the list in the designs, and I think it makes sense for a "continuous scroll" use-case. If you're talking about first on the knobs list - I'm not sure that it's really that important.